src/routes/committee/schedule/+page.svelte

```diff
@@ -64,13 +65,15 @@
 
 		let rehearsal = {
 			// TODO: Use current edition
+			id: newRehearsal.id,
```

Comment:
is dit undefined als het niet bestaat? Of het null? Want als het null is wil je waarschijnlijk undefined neerzetten, cuz idk wat firebase doet met het id als het null is

Reply:
Hm hij pakt hier het id dat ie net heeft gegenereerd voor t nieuwe doc, wss willen we t gewoon weglaten